lib/pliny.rb

```diff
@@ -8,6 +8,7 @@
 require_relative "pliny/helpers/params"
 require_relative "pliny/helpers/serialize"
 require_relative "pliny/log"
+require_relative "pliny/metrics/backends/logger"
```

Comment:
Any reason why this is required before `pliny/metrics`? Looks a little off having it in this order.